pipenv/environments.py

```diff
@@ -81,7 +81,15 @@ def normalize_pipfile_path(p):
 os.environ.pop("__PYVENV_LAUNCHER__", None)
 # Internal, to tell whether the command line session is interactive.
 SESSION_IS_INTERACTIVE = _isatty(sys.stdout)
-PIPENV_IS_CI = env_to_bool(os.environ.get("CI") or os.environ.get("TF_BUILD") or False)
+
+# TF_BUILD indicates to Azure pipelines it is a build step
+PIPENV_IS_CI = os.environ.get("CI") or os.environ.get("TF_BUILD")
```

Comment:
What is TF_BUILD used for? 
We should document this for the future.
One line above should suffice.

Reply:
I tried to do that in line 85.

Reply:
I missed that. Sorry.